webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -173,8 +173,15 @@ def update_response_overview_plot(
             response = ensemble.responses[selected_response]
 
             x_axis = response.axis
+
+            if x_axis is None:
```

Comment:
This was needed because mypy complained that `None` could neither have `get_loc` nor `index` methods in line [213](https://github.com/equinor/webviz-ert/pull/310/files#diff-51ada3feb545708cc141a34f1482d91f92630bed64d5501567bf3eae09c43fddR213)
This will make the [None condition below](https://github.com/equinor/webviz-ert/pull/310/files#diff-51ada3feb545708cc141a34f1482d91f92630bed64d5501567bf3eae09c43fddL178) to be outdated. 
But as I am not sure about the if-statement above, I did not remove the below.

Which situation we can have `None` as data type to `x_axis`?